steward/src/cork.rs

```diff
@@ -108,13 +198,14 @@ async fn build_cork(request: &SubmitRequest) -> Result<Cork, Error> {
     })
 }
 
+#[allow(unreachable_patterns)]
 fn get_encoded_call(data: CallData) -> Result<Vec<u8>, Error> {
     match data {
         AaveV2Stablecoin(call) => Ok(aave_v2_stablecoin::get_encoded_call(
             call.function
                 .expect("call data for Aave V2 Stablecoin cellar was empty"),
         )),
-        Uniswapv3Rebalance(params) => Ok(uniswapv3::get_encoded_call(params)),
+        _ => Err(ErrorKind::Http.context("empty call data for cellar").into()),
```

Comment:
Removing that variant will break the code. It's a bad practice to have one variant in an Enum. However, since we have only one cellar, there's nothing I can do than try to patch things up.

That warning originates because the compiler can't reach the Err arm. That's because the match section won't run into an error. The reason is because we have one cellar which is `Aavestablecoin` and we're telling the match section that `Aavestablecoin` call is `Ok`. Since we'll only get `Aavestablecoin` call for now, it'll always return true, with no error.

While the err match arm is useless, removing it will throw an error, since we can't have one match arm in a match pattern

Reply:
In my opinion, we'll remove this once we have another cellar. However, I'm open to suggestions on how to fix this if it'll cause a breach of security or break anything.

Reply:
> Removing that variant will break the code. It's a bad practice to have one variant in an Enum. However, since we have only one cellar, there's nothing I can do than try to patch things up.

Removing the `_` variant doesn't break anything, I just tested it.

Reply:
Damn! It kept saying "can't have a single match arm" until I ran cargo clean

Reply:
Weird. I wonder if there was an old version of the enum in the cache or something.